src/tests/ftest/osa/osa_online_extend.py

```diff
@@ -28,23 +28,19 @@ def setUp(self):
         """Set up for test case."""
         super().setUp()
         self.dmg_command = self.get_dmg_command()
-        self.ior_flags = self.params.get("ior_flags", '/run/ior/iorflags/*')
-        self.ior_apis = self.params.get("ior_api", '/run/ior/iorflags/*')
-        self.ior_test_sequence = self.params.get("ior_test_sequence",
-                                                 '/run/ior/iorflags/*')
-        self.ior_daos_oclass = self.params.get("obj_class",
-                                               '/run/ior/iorflags/*')
-        self.ior_dfs_oclass = self.params.get(
-            "obj_class", '/run/ior/iorflags/*')
+        self.daos_command = DaosCommand(self.bin)
```

Comment:
you can use self.get_daos_command() here, as you did for dmg above this line.

Reply:
Yes, Will do it when I re-enable the tests. Thanks.